projects/mac_ai/visualizations/llm_load_test/models/lts.py

```
@@ -59,12 +58,11 @@
     failures: int
 
 
-class KServeLLMPerformanceKPI(matbench_models.KPI, Settings): pass
-
-KServeLLMPerformanceKPIs = matbench_models.getKPIsModel("KServeLLMPerformanceKPIs", __name__, kpi.KPIs, KServeLLMPerformanceKPI)
+class KPI(matbench_models.KPI, Settings): pass
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)